bitnami/mongodb/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: mongodb
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/mongodb
-version: 16.0.3
+version: 16.0.4
```

As you are adding a new parameter, we need to bump the chart to a new minor version
```suggestion
version: 16.1.0
```